src/storage/src/hummock/mod.rs

```diff
@@ -159,6 +159,10 @@ impl HummockStorage {
         check_bloom_filter: bool,
         stats: &mut StoreLocalStatistic,
     ) -> HummockResult<Option<Option<Bytes>>> {
+        if check_bloom_filter {
```

Comment:
Can we move the bloom filter check logic into a common method so that it can be used by both get and prefix scan? We can update the metrics inside the method.